.github/workflows/linux.yml

```diff
@@ -481,6 +481,79 @@ jobs:
       runner: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores' }}
```

Comment:
I still see this structure:
![image](https://github.com/openvinotoolkit/openvino/assets/35459624/0324b9d9-9c79-410f-b32b-b3b2c3847efb)

Reply:
I moved it under common section